lib/spdy-transport/protocol/http2/parser.js

```diff
@@ -153,6 +166,9 @@ Parser.prototype.onFrameBody = function onFrameBody(header, buffer, callback) {
     this.onPriorityFrame(header, buffer, callback);
   else if (header.type === frameType.X_FORWARDED_FOR)
     this.onXForwardedFrame(header, buffer, callback);
+  else if (this._lastHeaderBlock)
+    return callback(this.error(constants.error.PROTOCOL_ERROR,
```

Comment:
Putting this `if` here will not really help, as far as I could tell, it should be moved to the very top, right after `HEADERS` and `CONTINUATION` handlers. `DATA` handler should be put after it.